prepare/formats/user_agent.py

```diff
@@ -1,9 +1,9 @@
 from src.unitxt.catalog import add_to_catalog
-from src.unitxt.formats import ICLFormat
+from src.unitxt.operators import ModelInputFormatter
 
-format = ICLFormat(
-    input_prefix="User: ",
-    output_prefix="Agent: ",
+format = ModelInputFormatter(
+    demo_format="User: {source}\nAgent:  {target}\n\n",
```

Comment:
Let's remove the extra space before the {target} (what used to be the target_prefix). It's was a mistake.

Reply:
Yes, now as `ICLFormat` gets down the stage, l will remove its heritage as well. Please see my reply [here](https://github.com/IBM/unitxt/pull/423#discussion_r1437459745), @yoavkatz